src/nextjs/index.ts

```diff
@@ -144,14 +151,64 @@ export class OttofellerNextjsProject extends NextJsTypeScriptProject {
     }
 
     // ANCHOR Set up Lighthouse audit
-    const lighthouse = options.lighthouse ?? true
+    const isLighthouseEnabled = options.isLighthouseEnabled ?? true
 
-    if (lighthouse) {
+    if (isLighthouseEnabled) {
       this.addDevDeps('@lhci/cli')
       this.addScripts({lighthouse: 'lhci autorun'})
       new SampleFile(this, 'lighthouserc.js', {sourcePath: path.join(assetsDir, 'lighthouserc.js')})
     }
 
+    // ANCHOR Set up Playwright
+    const isPlaywrightEnabled = options.isPlaywrightEnabled ?? true
+
+    if (isPlaywrightEnabled) {
+      this.addDeps('@playwright/test', 'playwright-qase-reporter')
+
+      this.addScripts({'test:e2e': 'playwright test'})
+
+      new SampleFile(this, 'playwright.config.ts', {sourcePath: path.join(assetsDir, 'playwright.config.ts.sample')})
```

Comment:
There is a `SampleDir` component that allows copying entire folders. I guess it might get the code more compact if we replace multiple `SampleFile` instances with a single `SampleDir`. Did you try it? Are there any problems with it?

Reply:
No, I didn't do that. Because I didn't know about this possibility. Thanks, I'll use it

Reply:
We concluded that the SampleDir won't work, since we need to have `sample` extension (in order to avoid additional TS errors on assets) and SampleDir can't rename files during copying, while copying files one by one allows for renaming and removing of the `sample` extension.
So we drop this issue.